stacks/dlaas/kubeflow/dlaas-tfjob/dlaas-bench/prototypes/dlaas-alexnet.jsonnet

```diff
@@ -0,0 +1,83 @@
+// @apiVersion 0.1
+// @name io.intel.pkg.dlaas-alexnet
+// @description dlaas multi-node benchmark
+// @shortDescription A multi-node DLaaS benchmark
+// @param name string Name for the job.
+
+local k = import "k.libsonnet";
+
+local name = params.name;
+local namespace = env.namespace;
+local image = "hub.docker.com/clearlinux/stacks-dlaas-kubeflow:latest";
```

Comment:
Is this going to exist eventually?

Reply:
Ah I think that this is just the image with benchmarks in it. I'm not really sure having this in the repo makes sense. I guess this demonstrates how to make the dlaas images work with k8s to some extent but it seems like it fits better with documentation than this repo.

Reply:
Agreed. once we get integrated with kubwflow, we can remove all this. This is to show the user how it will get laid out at the end.